demosplan/DemosPlanCoreBundle/Logic/Permission/AccessControlService.php

```diff
@@ -61,7 +59,11 @@ public function getPermissions(?OrgaInterface $orga, ?CustomerInterface $custome
         foreach ($roles as $roleName) {
             // Try to find an existing permission with the given parameters
 
-            $role = $this->getRoleByCode($roleName);
+            $role = $this->roleHandler->getRoleByCode($roleName);
+
+            if (null === $role) {
+                continue;
+            }
```

Comment:
minor personal note: I prefer to avoid jump actions like continue, because it increase the level of complexity (for me).
Instead you can wrap the whole logic into an if(null !== $role).
But I'm not sure if we as developers has this as best practice.

Reply:
I like this style more than nested ifs. For me this continue reduces cognitive complexity more deeply nested code

Reply:
what about discussing it in the team? I am with Sami and I am open to discuss it as well :+1: thanks for the remarks

Reply:
Yes, we could, maybe we should discuss this. In this case I'm okay because it is a conscious decision.